WoWPro_Leveling/Neutral/SL_Bastion.lua

```diff
@@ -1,12 +1,11 @@
 local guide = WoWPro:RegisterGuide('CagBastion', 'Leveling', "Bastion", 'Cagomei', 'Neutral')
```

Comment:
You should add in the copyright notice and we should check in a copy of it in GitHub